build.gradle

```diff
@@ -134,7 +134,7 @@ dependencies {
     implementation 'com.esotericsoftware.yamlbeans:yamlbeans:1.11'
     implementation "com.synopsys.integration:blackduck-common:${blackDuckCommonVersion}"
     implementation 'com.synopsys:method-analyzer-core:0.2.0'
-    implementation "${locatorGroup}:${locatorModule}:1.0.9"
+    implementation "${locatorGroup}:${locatorModule}:1.0.10"
```

Comment:
This Detect PR will be merged only after both CLL and dummy libs v1.0.10 are released and published.

Reply:
1.0.11*